pkg/operator/controller/certificate-publisher/publish_certs_test.go

```diff
@@ -100,15 +58,24 @@ func TestDesiredRouterCertsGlobalSecret(t *testing.T) {
 		defaultCert = newSecret("router-certs-default")
 		defaultCI   = newIngressController("default", "", "apps.my.devcluster.openshift.com")
 
-		ci1 = newIngressController("ci1", "s1", "dom1")
-		ci2 = newIngressController("ci2", "s2", "dom2")
-		s1  = newSecret("s1")
-		s2  = newSecret("s2")
-		// data has the PEM for defaultCert, s1, and s2 (which all have
-		// the same certificate and key).
-		data = bytes.Join([][]byte{
-			s1.Data["tls.crt"],
-			s1.Data["tls.key"],
+		defaultCIExplicitDefaultDefaultCert = newIngressController("default", "router-certs-default", "apps.my.devcluster.openshift.com")
```

Comment:
I suppose this is a typo.
```suggestion
		defaultCIExplicitDefaultCert = newIngressController("default", "router-certs-default", "apps.my.devcluster.openshift.com")
```

Reply:
It actually isn't; it's just confusing.  * grin *.  I've added a comment to explain:
```
		// defaultCIExplicitDefaultDefaultCert is an ingresscontroller
		// named "default" that specifies an explicit reference for
		// default certificate secret, where that secret is the same
		// default one that the operator generates when none is
		// specified (the "default default certificate").
```
Is the comment sufficient, or would it still be less confusing if I renamed the variable?

Reply:
It is clear now with the comment.

Reply:
Maybe renaming the variable would further aid clarity because I first wondered whether the run-on of DefaultDefault was intentional. And CI is Custom Ingress? Or Cluster Ingress? Maybe we could choose names like "production", or "staging", or "dev"...?

Reply:
I've renamed the variables: `defaultCIWithDefaultCertUnspecified`, `defaultCIWithDefaultCertSetToDefault`, and `defaultCIWithDefaultCertSetToCustom`.

Reply:
Hmm, I suppose while I'm renaming variables, I can get rid of the archaic "CI" initialism...  Done.